### PR TITLE
[CI:DOCS] Update podman build man page to match buildah bud man page

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: docs/source/markdown/podman-build.1.md [hunk 1]
```diff
@@ -353,11 +365,15 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
...
 Sets the configuration for network namespaces when handling `RUN` instructions.
-The configured value can be "" (the empty string) or "container" to indicate
-that a new network namespace should be created, or it can be "host" to indicate
-that the network namespace in which `podman` itself is being run should be
-reused, or it can be the path to a network namespace which is already in use by
-another process.
+
+Valid _mode_ values are:
+
+Valid _mode_ values are:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:371]: cut/paste error?  Line 369 and 371 look to be dupes

File: docs/source/markdown/podman-build.1.md [hunk 2]
```diff
@@ -353,11 +365,15 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
...
+
+- **none**: no networking;
+- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure;
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:374]:
```suggestion
- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure.
```

File: docs/source/markdown/podman-build.1.md [hunk 3]
```diff
@@ -353,11 +365,15 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
...
+- **ns:**_path_: path to a network namespace to join;
+- `private`: create a new namespace for the container (default)
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:376]: Just private?  I don't recall, but thought there were one or two more....

File: docs/source/markdown/podman-build.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-build.1.md:371]: Fixed

File: docs/source/markdown/podman-build.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/source/markdown/podman-build.1.md:374]: FIxed

File: docs/source/markdown/podman-build.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/source/markdown/podman-build.1.md:376]: Well there is private, none, host and ns.  There is also container: for podman run, but this does not work with build.

File: docs/source/markdown/podman-build.1.md [hunk 4]
```diff
@@ -353,11 +365,13 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
...
-that a new network namespace should be created, or it can be "host" to indicate
-that the network namespace in which `podman` itself is being run should be
-reused, or it can be the path to a network namespace which is already in use by
-another process.
+
+Valid _mode_ values are:
+
+- **none**: no networking;
+- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure.
+- **ns:**_path_: path to a network namespace to join;
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:373]: nit
```suggestion
- **ns:**_path_: path to a network namespace to join.
```

File: docs/source/markdown/podman-build.1.md [hunk 5]
```diff
@@ -353,11 +365,13 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
...
+- **private**: create a new namespace for the container (default)
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:374]: nit
```suggestion
- **private**: create a new namespace for the container (default).
```

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
This one is ready to go in.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

File: docs/source/markdown/podman-build.1.md [hunk 6]
```diff
@@ -188,6 +188,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--decryption-key** *key[:passphrase]*
+
+The [key[:passphrase]] to be used for decryption of images. Key can point to keys and/or certificates. Decryption will be tried with all keys. If the key is protected by a passphrase, it is required to be passed in the argument and omitted otherwise.
```
Comment (saschagrunert) [inline docs/source/markdown/podman-build.1.md:193]: For sake of consistency I recommend to keep the line break at 80 chars.

**Review by vrothberg — COMMENTED** [review]
Other than Sascha's comment, LGTM

**Comment by rhatdan** [comment]
Sadly I just destroyed this PR, will attempt to fix it.

File: docs/source/markdown/podman-build.1.md (hunk 6 above)
Comment (rhatdan) [reply docs/source/markdown/podman-build.1.md]: We don't have --decryption-key until the next vendor of buildah

File: docs/source/markdown/podman-build.1.md [hunk 7]
```diff
@@ -425,65 +476,77 @@ environment variable.  `export BUILDAH_RUNTIME=/usr/local/bin/runc`
...
-Set the create timestamp to seconds since epoch to allow for deterministic builds (defaults to current time).
-By default, the created timestamp is changed and written into the image manifest with every commit,
-causing the image's sha256 hash to be different even if the sources are exactly the same otherwise.
-When --timestamp is set, the created timestamp is always set to the time specified and therefore not changed, allowing the image's sha256 to remain the same. All files committed to the layers of the image will be created with the timestamp.
+Set the create timestamp to seconds since epoch to allow for deterministic
+builds (defaults to current time). By default, the created timestamp is changed
+and written into the image manifest with every commit, causing the image's
+sha256 hash to be different even if the sources are exactly the same otherwise.
+When --timestamp is set, the created timestamp is always set to the time
+specified and therefore not changed, allowing the image's sha256 to remain the
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:536]:
```suggestion
specified and therefore not changed, allowing the image's sha256 hash to remain the
```

File: docs/source/markdown/podman-build.1.md [hunk 8]
```diff
@@ -622,49 +685,65 @@ Only the current container can use a private volume.
...
-the mount propagation policy is set to `slave`, one way mount propagation is enabled
-and any mounts completed on the host for that volume will be visible only inside of the container.
-To control the mount propagation property of volume use the `:[r]shared`,
-`:[r]slave` or `:[r]private` propagation flag. The propagation property can
-be specified only for bind mounted volumes and not for internal volumes or
-named volumes. For mount propagation to work on the source mount point (mount point
-where source dir is mounted on) has to have the right propagation properties. For
-shared volumes, the source mount point has to be shared. And for slave volumes,
-the source mount has to be either shared or slave. <sup>[[1]](#Footnote1)</sup>
+inside container will not be visible on the host and vice versa. This behavior
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:715]:
```suggestion
inside containers will not be visible on the host and vice versa. This behavior
```
or perhaps "a container"

File: docs/source/markdown/podman-build.1.md [hunk 9]
```diff
@@ -784,21 +878,29 @@ mechanism:
...
-If you are using a useradd command within a Containerfile with a large UID/GID, it will create a large sparse file `/var/log/lastlog`.  This can cause the build to hang forever.  Go language does not support sparse files correctly, which can lead to some huge files being created in your container image.
+If you are using a useradd command within a Containerfile with a large UID/GID,
+it will create a large sparse file `/var/log/lastlog`.  This can cause the
+build to hang forever.  Go language does not support sparse files correctly,
+which can lead to some huge files being created in your container image.
 
-If you are using `useradd` within your build script, you should pass the `--no-log-init or -l` option to the `useradd` command.  This option tells useradd to stop creating the lastlog file.
+If you are using `useradd` within your build script, you should pass the
+`--no-log-init or -l` option to the `useradd` command.  This option tells
+useradd to stop creating the lastlog file.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-build.1.md:903]: This is also in the troubleshooting guide.  I think this makes sense to leave as is here, but maybe we should add a pointer to the troubleshooting guide here too?

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**